doc/parameters.md

```diff
@@ -51,6 +51,25 @@ Double colon (`:`) separated list of environments (or paths) APIcast should load
 It can be used instead of `-e` or `---environment` parameter on the CLI and for example
 stored in the container image as default environment. Any value passed on the CLI overrides this variable.
 
+### `APICAST_LOAD_SERVICES_WHEN_NEEDED`
+**Values:**
+- `true` or `1` for _true_
+- `false`, `0` or empty for _false_
+
+**Default:** _false_
+
+This option might be useful for users with many services configured. By default,
```

Comment:
```suggestion
This option is useful when there are many services configured. By default,
```

Reply:
I wrote "might" because it depends a lot on factors such as the number of services, the latency between apicast and the 3scale portal admin, the TTL of the config, etc. We cannot guarantee that this option will always work better than the default.

Reply:
Got you. This is important.
Suggestion:
`This option can be used when there are many services configured. However, its performance depends on additional factors such as the number of services, the latency between APIcast and the 3scale Admin Portal, the Time To Live (TTL) of the configuration, etc.` <better if you can include a line break before starting the next line with _By default_>